dive_authors.py

```diff
@@ -191,37 +145,85 @@ def get_authordateinfo_list(self):
 
         return self.authors_dateinfo
 
-    def get_latest_author_by_email(self, name):
-        if not hasattr(name, 'decode'):
-            name = str.encode(name)
 
-        name = name.decode("unicode_escape", "ignore")
-        return self.authors_by_email[name]
+def get_all_repos(access_token):
+    repos = []
+    page = 1
+    url = 'https://api.github.com/orgs/industrydive/repos?per_page=100&page=%s&access_token=%s'
+    while True:
+        response = requests.get(url % (page, access_token))
+
+        try:
+            response.raise_for_status()
+        except requests.exceptions.HTTPError as e:
+            print 'Error: %s' % e
```

Comment:
i think using `print` statements is generally frowned on. can we use some kind of logger and pipe it to stdout or something?

Reply:
Yah, using `logging` now